util/warnp.h

```diff
@@ -8,6 +8,8 @@
 #define warn libcperciva_warn
 #define warnx libcperciva_warnx
 
+#define WARNP_SYSLOG_MAX_LINE 4095
```

Comment:
`/* Maximum length of messages sent to syslog; longer warnings will be truncated. */`

Reply:
done